Logger/FileLogger.cs

```diff
@@ -0,0 +1,16 @@
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        public string filePath {get; set;}
```

Comment:
There is no reason for this to be public - it should be kept a private member of the class.
```suggestion
        private string _filePath;
```

Also, this line would be giving you a warning if nullability was enabled as it is a possible null reference.